tlsutil/config.go

```diff
@@ -252,7 +255,8 @@ func (c *Configurator) Update(config Config) error {
 	c.manual.cert = cert
 	c.manual.caPems = pems
 	c.caPool = pool
-	c.version++
+	atomic.AddUint64(&c.version, 1)
+	c.log("Update")
```

Comment:
This changes the behaviour slightly to only log `update` when it didn't error. Which is fine I guess.

Reply:
Correct, I should have mentioned that. Seems like only logging the update in the success case is slightly more correct, although these are trace level logs, so we probably don't see them very often.